adding cody chat context selection to docs (#61203)
sourcegraph/sourcegraph: doc/cody/capabilities/chat.md

  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/cody-write-code.mp4" type="video/mp4">
</video>

## Selecting Context

> Cody Chat allows you to provide context to your questions by selecting specific files, functions, or code blocks.

Reviewer: ```suggestion
Cody's chat allows you to add files and symbols as context in your messages.
```

Reviewer: and then I'd just have a short bullet list

```
- Type `@` and then a filename to include a file as context.
- Type `@#` and then a symbol name to include the symbol's definition as context. Functions, methods, classes, types, etc., are all symbols.
```

and delete the rest. I think people will get the idea.
